ed25519/suite_test.go

```diff
@@ -0,0 +1,63 @@
+package ed25519
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+var edSuite = NewAES128SHA256Ed25519(false)
+
+func TestCurvePointClone(t *testing.T) {
+	P1, _ := edSuite.Point().Pick(nil, edSuite.Cipher(nil))
+	P2 := P1.Clone()
+	assert.True(t, P1.Equal(P2))
+}
+
+func TestCurvePointCloneRandom(t *testing.T) {
+	if testing.Short() {
+		t.Skip("Randomized test skipped in short mode")
+	}
+	iters := 1000
+	null := edSuite.Point().Null()
+	for iters > 0 {
```

Comment:
OK, will change it together with putting all tests for each suite in `test/` at the end.